Sources/Controllers/2.0/OAMapSettingsOverlayUnderlayScreen.mm

```diff
@@ -469,7 +469,16 @@ - (UITableViewCell*) tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             [cell.switchView removeTarget:self action:NULL forControlEvents:UIControlEventValueChanged];
             if (indexPath.section == kMapVisibilitySection)
             {
-                [cell.switchView setOn: [[OARootViewController instance].mapPanel isOverlayUnderlayViewVisible]];
+                
+                if ([item[@"title"] isEqualToString:OALocalizedString(@"map_settings_show_slider_map")])
```

Comment:
Not clear - why do we need if-else here?
P.S. comparing localized strings - bad approach.